### PR TITLE
[circle-mlir/models] Add BatchNormalization

### PR DESCRIPTION
**Pull request by seanshpark**
This will add test models for onnx.BatchNormalization from nn.BatchNorm2d().

### CODE REVIEW

**Comment by seanshpark** [comment]
- for #15105
- from #15106

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by shs-park — COMMENTED** [review]
👍
